src/zen/websocket_server.cpp

```diff
@@ -898,6 +898,9 @@ class WsHandler
                     }
                 }
 
+                // Push the default value for fromAddress param.
+                cmdParams.push_back("*");
```

Comment:
This value must be changed when if the RPC command `sc_send_certificate()` is modified.